### DISCUSSION TITLE
Assigning a scratchpad to a display

### DISCUSSION THREAD

**Discussion by CrustyCode** [i3/i3#6076] [Q&A] [answered]
```
assign [title="scratchpad_ai"] output "DP-0"
assign [title="scratchpad_todo"] output primary
```

Neither of the above attempts are working. Is there a way to do this?

(Yes, xrandr shows that my DP-0 display is set to primary)
(i3 version 4.23 (2023-10-29))

EDIT:
Also not working: 
```
bindsym $mod+t [title="scratchpad_todo"] **focus output "DP-0"**, scratchpad show, move position center;
```

**Comment by CrustyCode** [comment, marked as answer]
Solved!

`bindsym $mod+t [title="scratchpad_todo"] scratchpad show, move container to output primary, move position center; focus output primary; \
`
The semi-colon before focusing is necessary, otherwise it crashes i3wm.

**Reply by orestisfl** [reply]
>  The semi-colon before focusing is necessary, otherwise it crashes i3wm.

yeah, that shouldn't happen :fearful: thanks for noticing
